test/adapters/ragequit.test.js

```diff
@@ -571,18 +577,16 @@ describe("Adapter - Ragequit", () => {
     const memberAUnits = await bank.balanceOf(memberA, UNITS);
     expect(memberAUnits.toString()).equal("10000000000000000");
 
-    await expect(
-      this.adapters.ragequit.ragequit(
-        this.dao.address,
-        memberAUnits,
-        toBN(0),
-        [], //empty token array
-        {
-          from: memberA,
-          gasPrice: toBN("0"),
-        }
-      )
-    ).to.be.revertedWith("missing tokens");
+    await this.adapters.ragequit.ragequit(
+      this.dao.address,
+      memberAUnits,
+      toBN(0),
+      [], //empty token array
+      {
+        from: memberA,
+        gasPrice: toBN("0"),
+      }
+    );
```

Comment:
```suggestion
    );
    expect((await bank.balanceOf(memberA, UNITS)).toString()).equal("0");
```

Reply:
Added the assert to make sure the units were set to zero after the ragequit operation without any tokens.